homework/kaydunov_denis/spring-aop/src/main/java/github/kaydunovdenis/App.java

```diff
@@ -0,0 +1,25 @@
+package github.kaydunovdenis;
+
+import github.kaydunovdenis.aspects.Logging;
+import github.kaydunovdenis.service.HumanService;
+import github.kaydunovdenis.service.UserServiceI;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class App {
+    public static void main(String[] args) {
+        ApplicationContext ctx = new ClassPathXmlApplicationContext("spring-config.xml");
+        UserServiceI userService = (UserServiceI) ctx.getBean("UserService");
+        HumanService humanService = (HumanService) ctx.getBean("HumanService");
+        Logging logging = (Logging) ctx.getBean("Log");
+
+        userService.createUser();
+        userService.updateUser();
+        try {
+            userService.riskyOperationOnUser();
+        } catch (Exception ex) {
+            System.out.println(ex.toString());
+        }
+        userService.simpleOperationOnUser("userName", "userNic");
```

Comment:
move this to test, 
try initialize ctx in test enviromnent. Maybe you could create a BaseClass that will load ctx and all others will extend it?